tests/test_trigger_bisect_jobs.py

```diff
@@ -66,6 +68,38 @@ def mocked_call(cmds, dry_run=False):
     "OPENQA_INVESTIGATE_ORIGIN=https://openqa.opensuse.org/tests/7848818",
 ]
 
+def test_catch_CalledProcessError(caplog):
+    import subprocess
+    args = args_factory()
+    args.url = "https://openqa.opensuse.org/tests/7848818"
+    openqa.fetch_url = MagicMock(side_effect=mocked_fetch_url)
+    cmd_args = ["openqa-clone-job",
+                "--skip-chained-deps",
+                "--json-output",
+                "--within-instance",
+                "https://openqa.opensuse.org/tests/7848818",
+                "OS_TEST_ISSUES=21770,21926,21954,22030,22077,22085,22192",
+                "TEST=foo:investigate:bisect_without_21637",
+                "OPENQA_INVESTIGATE_ORIGIN=https://openqa.opensuse.org/tests/7848818","_GROUP=0"]
```

Comment:
do we really need all of those parameters? When I tested it, it didn't make any difference.

Reply:
likely not